e2e/snapshot.css

```
@@ -0,0 +1,35 @@
+/* スクロールバーの非表示（OS間差分抑制） */
+::-webkit-scrollbar {
+  display: none !important;
+}
+
+html {
+  scrollbar-width: none !important;
+}
+
+/* 画像の遅延読み込みを即時読み込みに変更 */
+img[loading="lazy"] {
+  loading: eager !important;
```

Stylelint (17.3.0)

[error] Unexpected unknown property "loading" (property-no-unknown)

(property-no-unknown)